Sources/ParseLiveQuery/Internal/QueryEncoder.swift

```diff
@@ -16,14 +16,39 @@ import Parse
 extension Dictionary where Key: StringLiteralConvertible, Value: AnyObject {
     init(query: PFQuery) {
         self.init()
-
+        
         let queryState = query.valueForKey("state")
         if let className = queryState?.valueForKey("parseClassName") {
             self["className"] = className as? Value
         }
-
+        
         if let conditions: [String:AnyObject] = queryState?.valueForKey("conditions") as? [String:AnyObject] {
-            self["where"] = conditions as? Value
+            let newConditions = conditions.sanitizeParseObjects()
+            self["where"] = newConditions as? Value
         }
     }
 }
+
+extension Dictionary {
+    func sanitizeParseObjects() -> Dictionary {
+        var newSelf = Dictionary()
```

Comment:
I'm not sure I'm in love with the name of this function - since it's not mutating, would `var encodedQueryDictionary: Dictionary` be cleaner?